recipes/prefect/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "prefect" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
```

You could pull the sources from GitHub for now rather than using the wheel:
```suggestion
  url: https://github.com/PrefectHQ/{{ name }}/archive/{{ version }}.tar.gz
  
```